source/agora/common/API.d

```diff
@@ -90,4 +90,22 @@ public interface API
     ***************************************************************************/
 
     public PublicConfig getPublicConfig();
+
+    /***************************************************************************
+
+        Send message to this node
+
+        Parameter: 
+            string message(Hex)
+
+        Returns: 
```

Comment:
When a function is a `void` function, you don't have to document `Returns: void`, so you can remove this part.